### ISSUE TITLE
Presence of GPOS7 lookups

### ISSUE THREAD

**Issue by chrissimpkins** [googlefonts/googlesans#479]
New fontbakery warning as of revisions + fontbakery release update that occurred in #474:

```
    Ensure no GSUB5/GPOS7 lookups are present.
    with build/GoogleSans/variable/GoogleSans[GRAD,opsz,wght].ttf

      Rationale:                                                                
      Versions of fonttools >=4.14.0 (19 August 2020) perform an optimisation on
      chained contextual lookups, expressing GSUB6 as GSUB5 and GPOS8 and GPOS7 
      where possible (when there are no suffixes/prefixes for all rules in the  
      lookup).                                                                  
                                                                                
      However, makeotf has never generated these lookup types and they are rare 
      in practice. Perhaps before of this, Mac's CoreText shaper does not       
      correctly interpret GPOS7, and certain versions of Windows do not         
      correctly interpret GSUB5, meaning that these lookups will be ignored by  
      the shaper, and fonts containing these lookups will have unintended       
      positioning and substitution errors.                                      
                                                                                
      To fix this warning, rebuild the font with a recent version of fonttools. 

     WARN Font contains a GPOS7 lookup which is not processed by macOS          
          [code: has-gpos7]                                                     

     WARN Font contains a GSUB5 lookup which is not processed by macOS          
          [code: has-gsub5]                                                     


    Result: WARN
```

Fix:

- bump our fontmake compiler pipeline to use a new version of the fontTools library dependency

**Comment by chrissimpkins** [comment]
It looks like the changes in https://github.com/fonttools/fonttools/pull/2540 need to land in a release before we are able to address this.

**Comment by behdad** [comment]
> WARN Font contains a GSUB5 lookup which is not processed by macOS          
>           [code: has-gsub5]

This is not accurate. The macOS issue was only about GPOS7.

**Comment by chrissimpkins** [comment]
> > WARN Font contains a GSUB5 lookup which is not processed by macOS
> > [code: has-gsub5]
> 
> This is not accurate. The macOS issue was only about GPOS7.

Ty Behdad!  https://github.com/googlefonts/fontbakery/issues/3689

**Comment by chrissimpkins** [comment]
It sounds like the issue will be presence of GPOS7, not GSUB5.  The fonttools lib write and the fontbakery check are being updated to reflect this.  OK to ignore the warning on our builds for now.  We will implement the compiler update to address the new approach to GPOS7 after it rolls out.

**Comment by chrissimpkins** [comment]
This is addressed in #487

**Comment by chrissimpkins** [comment]
This was addressed in our pipeline updates merged in #487
